Scripts/Headers.lua

```diff
@@ -68,7 +68,6 @@ ScreenSelectMusicHeaderTexts = {
   '??????????????? okay',
   'cool! your point?',
   'https://streamable.com/noooo',
-  'conflict歌います。ズォールヒ～～↑wwwwヴィヤーンタースwwwwwワース フェスツwwwwwwwルオルwwwwwプローイユクwwwwwwwダルフェ スォーイヴォーwwwwwスウェン ネwwwwヤットゥ ヴ ヒェンヴガｒジョｊゴアｊガオガオッガwwwじゃｇｊｊ',
```

Comment:
this makes me want to add back the jp font

Reply:
Now reconvet due to f09ce36